File: swagger/models/get_client_indexes_request.py
from .base_model import BaseModel


class GetClientIndexesRequest(BaseModel):
	"""
	no description available

	Attributes:

		required_only
		When `true`, filters the results to only indexes that are required on
		creation.<br />  When `false` or omitted, returns all of the client
		indexes.
	"""

	input_map = {
		'RequiredOnly': 'required_only',
		}

	output_map = {
		'required_only': 'RequiredOnly',
		}

File: swagger/Models/GetClientIndexesRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientIndexesRequest extends BaseModel
{
	/**
	 * When `true`, filters the results to only indexes that are required on
	 * creation.<br />  When `false` or omitted, returns all of the client
	 * indexes.
	 * 
	 * @var boolean 
	 */
	public $requiredOnly;

	protected function getInputMap()
	{
		return [
			'RequiredOnly' => ['requiredOnly', 'boolean', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'requiredOnly' => ['RequiredOnly', 'boolean', null],
			];
	}
}
